lib/pbench/agent/tool_meister_start.py

```diff
@@ -1164,15 +1169,9 @@ def start(_prog: str, cli_params: Namespace) -> int:
 
         if orchestrate:
             try:
+                # FIXME:  Should we use ssh_path instead of ssh_cmd?
                 start_tms_via_ssh(
-                    prog.parent,
-                    ssh_cmd,
-                    Path(ssh_path),
-                    tool_group,
-                    ssh_opts,
-                    redis_server,
-                    redis_client,
-                    logger,
+                    prog.parent, ssh_cmd, tool_group, ssh_opts, redis_server, logger
                 )
```

Comment:
Thoughts, @portante?

Reply:
I think `ssh_cmd` should be dropped in favor of `ssh_path`, which should not be a `Path` object, but just the return value from `shutil.which("ssh")`.  Then as mentioned elsewhere, fix the `TempalteSsh()` constructor to accept a `str`.

Reply:
Given the confusion here (i.e., in general, around our use of "path-like" values), I would prefer not to use the name `ssh_path` to refer to something which is not a `Path` object.  So, would it be OK to stick with a variable named `ssh_cmd` which is a `str`, and get rid of the `Path` object?  (Or, could you propose another name?)

Reply:
`ssh_cmd` is fine.